app/controllers/v3/service_credential_bindings_controller.rb

```diff
@@ -65,10 +73,18 @@ def serialized(message)
     Presenters::V3::ServiceCredentialBindingPresenter.new(service_credential_binding, decorators: decorators(message)).to_hash
   end
 
+  def credential_binding_details
+    Presenters::V3::ServiceCredentialBindingDetailsPresenter.new(service_credential_binding).to_hash
+  end
+
   def ensure_service_credential_binding_is_accessible!
     not_found! unless service_credential_binding_exists?
   end
 
+  def ensure_binding_details_are_accessible!
```

Comment:
I'm generally a bit fan of using small functions to reduce cognitive load. For this function, my first thought was "what does accessible mean?", and I actually find the implementation easier to understand than the function name. But that's just my opinion and other people might feel differently.